### DISCUSSION TITLE
" You have selected base model other than SDXL. This is not supported yet." Will this be fixed in the future?

### DISCUSSION THREAD

**Discussion by Woukim** [lllyasviel/Fooocus#871] [Q&A]
![image](https://github.com/lllyasviel/Fooocus/assets/138323639/193c0fe5-77ee-468f-a5c8-9ad658f51e0f)

**Comment by AkoZ** [comment]
you can also try to use SD15 in fooocus as a refiner .. with **Refiner Switch** (use advanced mode) in Model tab, 
* Use 0.4 for SD1.5 realistic models; or 0.667 for SD1.5 anime models; or 0.8 for XL-refiners; or any value for switching two SDXL models.
* * with some tries , I'm not really convinced of "like SD15" mode, but try again ?

**Comment by murat-gunay** [comment]
@lllyasviel hi. First, I appriciate with your great effort. My quesiton is that: will we be able to use other sd architectures such as [Segmind SSD-1B](https://blog.segmind.com/introducing-segmind-ssd-1b/amp/) or [Pixart-alpha](https://arxiv.org/pdf/2310.00426.pdf) in Fooocus

**Reply by mel870** [reply]
Segmind SSD-1B works as I've tested this myself. Simply place it in the checkpoint folder. So far it appears compatible with image prompting. But it doesn't work with PyraCanny or cpds. However, Pixart-alpha  needs a 24GB GPU unless I'm misunderstanding and that's just requirements for training.

**Reply by recoilme** [reply]
i download ssd-1b in diffusers format and put in models/diffusers folder, but i didnt see model in menu(

**Reply by mel870** [reply]
You have to place it in models/checkpoints folder.

**Reply by rubenmejiac** [reply]
I tested SSD-1B.safetensors in Fooocus version 2.1.823, and it works as intended, with speed and quality sampling settings.
The "extreme speed" setting does not work (Fooocus implementation of LCM), but it compensates that the generation speeds are much faster than those of standard SDXL models.
Image prompting works, variations and upscale also, outpainting/inpainting does not.

**Comment by g2asell2019** [comment]
same question, why only support sd15 as refiner but not as base, simply just swap to two types

**Comment by StanleyPain** [comment]
Would love to see base SD1.5 support as well.

**Comment by R4290N** [comment]
same question, pls add  base SD1.5 support
